src/main/java/ca/wolfram/beta/symath/operations/AddNode.java

```diff
@@ -35,19 +35,49 @@ public NodeType getType() {
         return NodeType.ADD;
     }
 
+    /**
+     * Extracts all constant integers and appends it as one {@link ca.wolfram.beta.symath.base.ConstantNode} if the sum != 0
+     * Flattens children if they are also of type ADD
+     *
+     * @return true is isConstant, false otherwise
+     */
     @Override
     public boolean simplify() {
```

Comment:
I feel like this could be in the scope of this PR. but we can have another one in the future